pkg/operation/botanist/component/extensions/dnsrecord/dnsrecord.go

```diff
@@ -275,3 +279,13 @@ func (c *dnsRecord) SetRecordType(recordType extensionsv1alpha1.DNSRecordType) {
 func (c *dnsRecord) SetValues(values []string) {
 	c.values.Values = values
 }
+
+func (c *dnsRecord) ValuesHaveChanged() bool {
```

Comment:
This method should not be exported and the name could better reflect what it does.
```suggestion
func (c *dnsRecord) valuesDontMatchDNSRecord() bool {
```

Reply:
done.